### DISCUSSION TITLE
Framework gutenberg playground, BlockTools cursor issue.

### DISCUSSION THREAD

**Discussion by kohheepeace** [WordPress/gutenberg#57138] [Q&A]
@youknowriad 

## Assumption
I have started a gutenberg project for framework(standalone) with the following referenced Links. In doing so, I encountered a problem, so I would like to ask a question.

📍 stackblitz URL
https://stackblitz.com/edit/stackblitz-starters-papfri

### Referenced links to setup project
- https://github.com/WordPress/gutenberg/blob/trunk/storybook/stories/playground/fullpage/index.js
- [platform docs#intro "Bootstrap your block editor"](https://github.com/WordPress/gutenberg/blob/cdeca67b63635cca295ba8f4874130c53f394ab0/platform-docs/docs/intro.md?plain=1#L33)

## Problems
As shown in the following video, **it takes two clicks to edit paragraph.**

The first click places the focus on the BlockTools button, and the second click makes it editable.

https://github.com/WordPress/gutenberg/assets/29557494/f24f6377-8e2b-470a-9604-7d7210c6b5aa

**Comment by youknowriad** [comment]
Thanks for creating this discussion. The stackblitz link is very helpful. I've converted this to a bug issue here #57139 and I'm closing this discussion.

**Reply by kohheepeace** [reply]
Thanks for quick checking!
